vnext/Microsoft.ReactNative/Views/TextViewManager.cpp

```diff
@@ -290,4 +317,20 @@ TextTransform TextViewManager::GetTextTransformValue(ShadowNodeBase *node) {
   return TextTransform::Undefined;
 }
 
+void TextViewManager::AddToPressableCount(ShadowNodeBase *node, int pressableCount) {
+  if (!std::wcscmp(node->GetViewManager()->GetName(), GetName())) {
+    const auto textNode = static_cast<TextShadowNode *>(node);
+    textNode->pressableCount += pressableCount;
+  }
+}
+
+int64_t TextViewManager::GetReactTagAtPoint(ShadowNodeBase *node, const winrt::Point &point) {
+  if (!std::wcscmp(node->GetViewManager()->GetName(), L"RCTText")) {
+    const auto textNode = static_cast<TextShadowNode *>(node);
+    return textNode->GetReactTagAtPoint(point);
+  }
+
+  return node->m_tag;
+}
```

Comment:
this got me thinking... a community module View Manager would not be able to implement this functionality, as the logic to do custom hit testing and find internal react tags is tightly coupled with system internals.
If I got that right, I'd love to see this be made accessible to 3rd parties to plug into (via a new `IViewManagerWithHitTesting` or similar interface). It doesn't need to be part of this PR, but if we can get it done for 3rd parties, it would be cool if Text used the public API and be decoupled from the internals if it doesn't need to be.

Reply:
e.g. this would also enable TextBlock etc. from react-native-xaml to opt into this kind of logic too if needed

Reply:
Totally agree. In the C# days, we had `IReactCompoundView` for exactly this reason: https://github.com/microsoft/react-native-windows/blob/0.59-stable/current/ReactWindows/ReactNative.Shared/UIManager/IReactCompoundView.cs

Reply:
Not going to fix in this PR.